clifford/test/test_clifford.py

```diff
@@ -386,6 +386,8 @@ def test_binary_op_preserves_dtype(self, dtype, func, g3):
         operator.pos,
```

Comment:
I think you should be able to add `operator.and_,  # vee product` on line 375 now too.